docs/DeveloperGuide.md

```diff
@@ -272,21 +272,27 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 | Priority | As a …​        | I want to …​                                                                            | So that I can…​                                                                                             |
```

Comment:
Good job tidying up the user stories! I just have a small nitpick - perhaps one more can be included regarding the `help` command?

Reply:
Yup okay! Added and pushed